grid2op/tests/test_RulesByArea.py

```diff
@@ -0,0 +1,146 @@
+# Copyright (c) 2023, RTE (https://www.rte-france.com)
+# See AUTHORS.txt
```

Comment:
I just noticed it here, did you add yourself in the AUTHORS.txt ?

Reply:
Nope, thanks for notifying !